db/schema.rb

```diff
@@ -393,16 +390,16 @@
     t.uuid "customer_id"
     t.float "vat_rate", default: 0.0, null: false
     t.bigint "credit_amount_cents", default: 0, null: false
-    t.string "credit_amount_currency"
     t.integer "status", default: 1, null: false
     t.string "timezone", default: "UTC", null: false
     t.integer "payment_attempts", default: 0, null: false
     t.boolean "ready_for_payment_processing", default: true, null: false
     t.uuid "organization_id", null: false
     t.integer "version_number", default: 2, null: false
+    t.string "currency"
     t.bigint "fees_amount_cents", default: 0, null: false
-    t.bigint "credit_notes_amount_cents", default: 0, null: false
     t.bigint "coupons_amount_cents", default: 0, null: false
+    t.bigint "credit_notes_amount_cents", default: 0, null: false
```

Comment:
Seems not related to that PR

Reply:
Well, I worked a lot with migration, and the order was a bit mixed :)
But if a drop create and remigrate my DB this is what I get so it will be fixed by merging one of the two PR having this issue.